code/modules/admin/topic.dm

```diff
@@ -2289,6 +2289,214 @@
+
+		// Create the reply message
+		if(sendername)
+			P.name = "[sendername]- [customname]"
+		else
+			P.name = "[customname]"
+		P.info = input_text
+		P.update_icon()
+		P.x = rand(-2, 0)
+		P.y = rand(-1, 2)
+		//P.offset_x += P.x
+		//P.offset_y += P.y
+
+		if(destination != "All Departments")
+			if(!fax.receivefax(P))
+				to_chat(src.owner, "<span class='warning'>Message transmission failed.</span>")
+				return
+		else
+			for(var/thing in GLOB.allfaxes)
+				var/obj/machinery/photocopier/faxmachine/F = thing
+				if(F.z in GLOB.station_z_levels)
+					addtimer(CALLBACK(src, .proc/handle_sendall, F, P), 0)
+
+		var/datum/fax/admin/A = new /datum/fax/admin()
+		A.name = P.name
+		A.from_department = faxtype
+		if(destination != "All Departments")
+			A.to_department = fax.department
+		else
+			A.to_department = "All Departments"
+		A.origin = "Administrator"
+		A.message = P
+		A.reply_to = reply_to
+		A.sent_by = usr
+		A.sent_at = world.time
+
+		to_chat(src.owner, "<span class='notice'>Message transmitted successfully.</span>")
+		if(notify == "Yes")
+			var/mob/living/carbon/human/H = sender
+			if(istype(H) && H.stat == CONSCIOUS && (istype(H.ears, /obj/item/device/radio/headset)))
+				to_chat(sender, "Your headset pings, notifying you that a reply to your fax has arrived.")
+		if(sender)
+			log_admin("[key_name(src.owner)] replied to a fax message from [key_name(sender)]: [input_text]")
+			message_admins("[key_name_admin(src.owner)] replied to a fax message from [key_name_admin(sender)] (<a href='?_src_=holder;[HrefToken(TRUE)];AdminFaxView=[REF(P)]'>VIEW</a>).", 1)
+		else
+			log_admin("[key_name(src.owner)] sent a fax message to [destination]: [input_text]")
+			message_admins("[key_name_admin(src.owner)] sent a fax message to [destination] (<a href='?_src_=holder;[HrefToken(TRUE)];AdminFaxView=[REF(P)]'>VIEW</a>).", 1)
+		return
+
+	else if(href_list["refreshfaxpanel"])
+		if(!check_rights(R_ADMIN))
+			return
+
+		fax_panel(usr)
+		return
+
+	else if(href_list["EvilFax"])
+		if(!check_rights(R_ADMIN))
+			return
+		var/mob/living/carbon/human/H = locate(href_list["EvilFax"])
+		if(!istype(H))
+			to_chat(usr, "This can only be used on instances of type /mob/living/carbon/human")
+			return
+		var/etypes = list("Borgification","Corgification","Death By Fire","Demotion Notice")
+		var/eviltype = input(src.owner, "Which type of evil fax do you wish to send [H]?","Its good to be baaaad...", "") as null|anything in etypes
+		if(!(eviltype in etypes))
+			return
+		var/customname = input(src.owner, "Pick a title for the evil fax.", "Fax Title") as text|null
+		if(!customname)
+			customname = "paper"
+		var/obj/item/paper/evilfax/P = new /obj/item/paper/evilfax(null)
+		var/obj/machinery/photocopier/faxmachine/fax = locate(href_list["originfax"])
+
+		P.name = "Central Command - [customname]"
+		P.info = "<b>You <i>really</i> should've known better.</b>"

```

Comment:
I'd rather this be done with `span` any reason it isn't?

Reply:
This is what is written on the paper. IIRC you can't use spans for that but correct me if I'm wrong.

Reply:
Span works here, player inputs to papers are sanitized.
Not code inputs, though.